example/main.go

```diff
@@ -45,7 +45,10 @@ func addKeyHandler(g *gossipdb.GossipDb) http.HandlerFunc {
 		if err != nil {
 			fmt.Println(err)
 		}
-		g.Set(body)
+		var data keyValue
```

Comment:
How about killing `keyValue` struct and using `Pair`struct declared below directly ?

Reply:
I think keeping different struct will reduce confusions. So we can keep it for the time being.